advanced/Scripts/piholeCheckout.sh

```diff
@@ -134,72 +138,75 @@ checkout() {
 
   if [[ "${1}" == "dev" ]] ; then
     # Shortcut to check out development branches
-    echo "::: Shortcut \"dev\" detected - checking out development / devel branches ..."
-    echo "::: Pi-hole core"
+    echo -e "  ${INFO} Shortcut \"dev\" detected - checking out development / devel branches..."
+    echo -e "  ${INFO} Pi-hole core"
     fetch_checkout_pull_branch "${PI_HOLE_FILES_DIR}" "development" || { echo "Unable to pull Core developement branch"; exit 1; }
     if [[ ${INSTALL_WEB} == "true" ]]; then
-      echo "::: Web interface"
+      echo -e "  ${INFO} Web interface"
       fetch_checkout_pull_branch "${webInterfaceDir}" "devel" || { echo "Unable to pull Web development branch"; exit 1; }
     fi
-    echo "::: done!"
+    echo "done!"
   elif [[ "${1}" == "master" ]] ; then
     # Shortcut to check out master branches
-    echo "::: Shortcut \"master\" detected - checking out master branches ..."
-    echo "::: Pi-hole core"
+    echo -e "  ${INFO} Shortcut \"master\" detected - checking out master branches..."
+    echo -e "  ${INFO} Pi-hole core"
     fetch_checkout_pull_branch "${PI_HOLE_FILES_DIR}" "master" || { echo "Unable to pull Core master branch"; exit 1; }
     if [[ ${INSTALL_WEB} == "true" ]]; then
-      echo "::: Web interface"
-      fetch_checkout_pull_branch "${webInterfaceDir}" "master" || { echo "Unable to pull web master branch"; exit 1; }
+      echo -e "  ${INFO} Web interface"
+      fetch_checkout_pull_branch "${webInterfaceDir}" "master" || { echo "Unable to pull AdminLTE master branch"; exit 1; }
     fi
-    echo "::: done!"
+    echo "done!"
+     
   elif [[ "${1}" == "core" ]] ; then
-    echo -n "::: Fetching remote branches for Pi-hole core from ${piholeGitUrl} ... "
+    str="Fetching branches from ${piholeGitUrl}"
+    echo -ne "  ${INFO} $str"
     if ! fully_fetch_repo "${PI_HOLE_FILES_DIR}" ; then
-      echo "::: Fetching all branches for Pi-hole core repo failed!"
+      echo -e "  ${CROSS} $str"
       exit 1
     fi
     corebranches=($(get_available_branches "${PI_HOLE_FILES_DIR}"))
```

Comment:
If this function fails, it will return the following to the user (and I'm not sure how to work around this, off the top of my head)

```
[i] Fetching branches from https://github.com/pi-hole/pi-hole.gitfatal: unable to access 'https://github.com/pi-hole/pi-hole.git/': Could not resolve host: github.com
```